03-task-manager/starter/quizeAnswers.txt

```diff
@@ -0,0 +1,8 @@
+1 middleware functions have access to the objects req and res and also the next function. 
+we can modify req or res using middleware, that makes it a powerfull tool. 
+next function calling for the next function - always must end our middleware with next()
+
+2CRUD stands for Create Read Update Delete/Destroy
+
+3   1. when using node.js need to install 'dotenv' package and add require('dotenv').config() in the file where we need access to .env - it gives access to .env file
```

Comment:
require('dotenv').config() will attach the object created from key and values of the .env file from .env file to process module's env object and hence you can access that value like process.env.PORT or process.env.password etc. Porcess is a file . . try it to print out console..log(process). It is jumbo file/module and one of the key in that module is env. You guessed it correctly. Just for fun...

Reply:
thanks for the clarification @Indra12345  will do!